2022/distributed-tracing.html

```diff
@@ -190,6 +190,7 @@ <h3 id="out-of-scope">Out of Scope</h3>
 
           <ul class="out-of-scope">
             <li>definition of (programming) language specific APIs for performance data collection; </li>
+            <li>browsers built-in implementation of trace headers propagation; </li>
```

Comment:
_nitpick:_ Is this referring to the "built-in implementation ..." _of_ browsers? That is, should it have a apostrophe to indicate genitive?
```suggestion
            <li>browsers' built-in implementation of trace headers propagation; </li>
```